app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "123456789012",
+    "firebase_url": "https://oppia-android-51g85.firebaseio.com",
+    "project_id": "oppia-android-51g85",
+    "storage_bucket": "oppia-android-51g85.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:123456789012:android:d544ded26ae57f7fe6e25b",
```

Comment:
I think we should make it obvious that this is fake data..something like "1:1234...:android:fakestring" or something
From this it doesn't look very obvious

Reply:
Done